finder-app/makefile

```diff
@@ -0,0 +1,22 @@
+# /***************************************************************************
+#  * AESD Assignment 2
+#  * Author: Chinmay Shalawadi 
+#  * Institution: University of Colorado Boulder
+#  * Mail id: chsh1552@colorado.edu
+#  * References: Stack Overflow, Man Pages
+#  ***************************************************************************/
+
+make:
+# Check if cross compile variable matches for ARM 
+ifeq ($(CROSS_COMPILE),aarch64-none-linux-gnu-)
```

Comment:
@neohere97 see the suggestions in lecture about how to define CC here for compatibility with future assignments.